src/calculator/routes.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express'
+import { textCredibility, twitterUserCredibility } from './service'
+
+const calculatorRoutes = express.Router()
+
+calculatorRoutes.get('/plain-text', function(req, res) {
+  res.send(textCredibility(req.query.text, {
+    weightBadWords: +req.query.weightBadWords,
+    weightMisspelling: +req.query.weightMisspelling,
+    weightSpam: +req.query.weightSpam
+  }))
+})
+
+calculatorRoutes.get('/twitter/social/:userId', function(req, res, next) { 
```

Comment:
Here, it should be /twitter/user/:userId, not social

Reply:
Because you change my code

Reply:
La loca @martinezfabiola